IntegrationDevelopment/I320/REST.postman_collection.json

```diff
@@ -0,0 +1,1301 @@
+{
+	"info": {
+		"_postman_id": "051c8a69-4de6-4c7a-99ce-87a492284f0f",
+		"name": "MyStoreIntegration_DataRetrieval",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Part 1: Authorizing the Application to Work with the Web Service",
+			"item": [
+				{
+					"name": "Lesson 1.2: Requesting token manually",
+					"request": {
+						"auth": {
+							"type": "noauth"
+						},
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"name": "Content-Type",
+								"type": "text",
+								"value": "application/x-www-form-urlencoded"
+							}
+						],
+						"body": {
+							"mode": "urlencoded",
+							"urlencoded": [
+								{
+									"key": "grant_type",
+									"value": "password",
+									"type": "text"
+								},
+								{
+									"key": "username",
+									"value": "admin",
+									"type": "text"
+								},
+								{
+									"key": "password",
+									"value": "123",
+									"type": "text"
+								},
+								{
+									"key": "scope",
+									"value": "api",
+									"type": "text"
+								},
+								{
+									"key": "client_id",
+									"value": "2CC9CB98-1292-B2CE-2EEF-23844235E9B2@MyStore",
+									"type": "text"
+								},
+								{
+									"key": "client_secret",
+									"value": "8xZkcLA_X9wMBoXjvcrrtw",
+									"type": "text"
+								}
+							]
+						},
+						"url": {
+							"raw": "https://localhost/MyStoreInstance/identity/connect/token",
+							"protocol": "https",
+							"host": [
+								"localhost"
+							],
+							"path": [
+								"MyStoreInstance",
+								"identity",
+								"connect",
+								"token"
+							]
+						}
+					},
+					"response": [
+						{
+							"name": "Lesson 1.2: Requesting token manually",
```

Comment:
Please unify capitalization in names. You have title-style capitalization in some names, and sentence-style in others.